diff --git a/node_modules/popup-ui/src/basic/Popup/index.js b/node_modules/popup-ui/src/basic/Popup/index.js
index 921daa0..2019f48 100644
--- a/node_modules/popup-ui/src/basic/Popup/index.js
+++ b/node_modules/popup-ui/src/basic/Popup/index.js
@@ -158,7 +158,7 @@ const styles = StyleSheet.create({
 		zIndex: 99999,
 		width: WIDTH,
 		height: HEIGHT,
-		backgroundColor: 'rgba(0, 0, 0, 0.5)',
+		backgroundColor: 'white',
 		alignItems: 'center',
 		top: 0,
 		left: 0
@@ -167,8 +167,10 @@ const styles = StyleSheet.create({
 		maxWidth: 300,
 		width: 230,
 		minHeight: 300,
-		backgroundColor: '#fff',
+		backgroundColor: 'black',
 		borderRadius: 30,
+		borderColor:"darkorange",
+		borderWidth:0.5,
 		alignItems: 'center',
 		overflow: 'hidden',
 		position: 'absolute',
@@ -180,7 +182,7 @@ const styles = StyleSheet.create({
 	Header: {
 		height: 230,
 		width: 230,
-		backgroundColor: '#FBFBFB',
+		backgroundColor: 'black',
 		borderRadius: 100,
 		marginTop: -120
 	},
@@ -193,7 +195,7 @@ const styles = StyleSheet.create({
 	Title: {
 		fontWeight: 'bold',
 		fontSize: 18,
-		color: '#333'
+		color: 'darkorange'
 	},
 	Desc: {
 		textAlign: 'center',
@@ -204,16 +206,17 @@ const styles = StyleSheet.create({
 		borderRadius: 50,
 		height: 40,
 		width: 130,
+		backgroundColor:"darkorange",
 		justifyContent: 'center',
 		alignItems: 'center',
 		marginTop: 30
 	},
 	TextButton: {
-		color: '#fff',
+		color: 'white',
 		fontWeight: 'bold'
 	},
 	Success: {
-		backgroundColor: '#AAF577',
+		backgroundColor: 'darkorange',
 		shadowColor: "#AAF577",
 		shadowOffset: {
 			width: 0,
